From 50a0829febf8825004e7d8a3b7975ca90cbb1edc Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Wed, 22 Jul 2020 13:28:36 +0700
Subject: [PATCH 254/504] ARM: dts: r8a7744: Add tddmac module

This patch support Renesas tddmac kernel module probe
(Image Extraction Direct Memory Access Controller)

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 568186d..8c89358 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -856,6 +856,14 @@
 			status = "disabled";
 		};
 
+		tddmac@0xfea00000 {
+			compatible = "renesas,vspm-tddmac","renesas,vspm";
+			reg = <0 0xfea00000 0 0x200>;
+			interrupts = <0 285 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7744_CLK_2DDMAC>;
+			power-domains = <&sysc R8A7744_PD_ALWAYS_ON>;
+		};
+
 		hscif0: serial@e62c0000 {
 			compatible = "renesas,hscif-r8a7744",
 				     "renesas,rcar-gen2-hscif", "renesas,hscif";
-- 
2.7.4

